project/pom.xml

```diff
@@ -1375,5 +1375,32 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>jenkins</id>
```

Comment:
@nacx Just out of curiosity: is the `jenkins` profile enabled automagically, or do we need to do that in the job configuration somewhere?